diff --git a/LogView.qml b/LogView.qml
index 4244e22..aa74e9a 100644
--- a/LogView.qml
+++ b/LogView.qml
@@ -17,7 +17,7 @@ Rectangle {
                    }
 
             text: {
-                var ts_string = "<" + timestamp.toLocaleString(Qt.locale(), "--MM-dd HH:mm") + "> ";
+                var ts_string = "[" + timestamp.toLocaleString(Qt.locale(), "MM-dd HH:mm") + "] ";
                 var suffix = "";
 
                 switch (status) {
